src/w2grid.js

```diff
@@ -5076,6 +5076,12 @@ class w2grid extends w2base {
                 let display
                 if (Array.isArray(sd.value)) {
                     display = `<span class="grid-search-count">${sd.value.length}</span>`
+                } else if (sf && sf.type == 'list') {
+                    const sd_value = sd.value
```

Comment:
I do not see a point to define constants here, why not 
```js
display = !!sd.text && sd.text !== sd.value ? `: ${sd.text}` : `: ${sd.value}`
```

Reply:
Well, you absolutely right, updated PR here.